### DISCUSSION TITLE
请问生医药的图注要求特殊格式该如何写？

### DISCUSSION THREAD

**Discussion by jvjvj02** [tuna/thuthesis#923] [Q&A]
生医药的要求的图注除了图序和图标题以外，后面还要另起一段描述性的文字，字号比图标题小一号，这个该怎么办呢？
![屏幕截图 2024-02-23 004849](https://github.com/tuna/thuthesis/assets/38560611/5c3c8192-e9c4-4e54-8f9b-87937e12aaf0)

**Comment by zepinglee** [comment]
目前 [`medical`](https://github.com/tuna/thuthesis/tree/medical) 分支有 `\figurenote` 和 `\tablenote` 命令。我会尽快合并到 `master` 分支。

另外我在 <https://github.com/tuna/thuthesis/issues/909#issuecomment-1919286550> 有些疑问。

**Reply by jvjvj02** [reply]
谢谢大佬，想问下在overleaf平台的THUthesis模板中要如何使用这个 medical 分支呢？是需要替换一些文件吗？

关于 https://github.com/tuna/thuthesis/issues/909#issuecomment-1919286550 里面的两个问题，我这里仅代表生医药的生物学博士毕业论文，医学实验班的不太清楚
1. 参考文献要求按照cell杂志2023年以前的格式，不需要加序号
2. 我们没有这项规定，都是用中文书写

**Comment by jvjvj02** [comment]
[2024春季论文格式说明.pdf](https://github.com/tuna/thuthesis/files/14381904/2024.pdf)
